src/js/controller.js

```diff
@@ -95,6 +95,10 @@ export default class Controller extends Observable {
         }
         this.triggerEvent("printProfile", filename)
     }
+	
+	onRemoveMapChip = (element) =>{
+		element.remove();
```

Comment:
I think this line should be in mapchip.js - i.e. the element should know how to hide itself - the event is triggered only in case someone is listening